src/EditorFeatures/CSharpTest2/Recommendations/UShortKeywordRecommenderTests.cs

```diff
@@ -761,5 +761,13 @@ class C
 {
     delegate*$$");
         }
+
+        [WorkItem(53585, "https://github.com/dotnet/roslyn/issues/53585")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterStaticLocalFunction()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(@"
+static $$"));
```

Comment:
so, you had all the tests after 'static' in a method, can you do the same for after 'async'? as well?  or make all the tests theories and test `static` `async` `async static` and `static async`.  Thanks!

Reply:
I used a `ClassData` theory added in b46098a and applied in f686b67.